src/server/game/Server/WorldSession.cpp

```diff
@@ -316,6 +309,7 @@ bool WorldSession::Update(uint32 diff, PacketFilter& updater)
         ClientOpcodeHandler const* opHandle = opcodeTable[opcode];
 
         METRIC_DETAILED_TIMER("worldsession_update_opcode_time", METRIC_TAG("opcode", opHandle->Name));
+        LOG_DEBUG("network", "message id {} under READ", opcode);
```

Comment:
I think need use `GetOpcodeNameForLogging(static_cast<OpcodeClient>(packet->GetOpcode()))`

Reply:
The point of logging the ID is efficiency.

Reply:
I would prefer the actual OP code name to be sent here. For convenience